### PR TITLE
ED-6428 : Update Collapsible component to receive field's formData

### PR DESCRIPTION
**Pull request by yusuff-rxnt**
Update Collapsible component to handle receive field's form data and formcontext.

### CODE REVIEW

File: src/CollapsibleField.js [hunk 1]
```diff
@@ -211,6 +219,8 @@ class CollapsibleField extends Component {
               onChange={formData => {
                 onBlur(formData);
               }}
+              formContext={{ ...formContext }}
```
Comment (sneu012) [inline src/CollapsibleField.js:222]:
```
formContext={formContext }
formData={formData }
```
Would the above make more sense?

Comment (ayjaweems) [reply src/CollapsibleField.js:222]: @sneu012 Yeah I think the change you commented makes more sense

File: src/CollapsibleField.js [hunk 2]
```diff
@@ -281,21 +291,21 @@ CollapsibleField.propTypes = {
...
-    }).isRequired,
+      wrapClassName: PropTypes.string
+    }).isRequired
   }).isRequired,
   registry: PropTypes.shape({
-    fields: PropTypes.object.isRequired,
-  }).isRequired,
+    fields: PropTypes.object.isRequired
+  }).isRequired
 };
```
Comment (ayjaweems) [inline src/CollapsibleField.js:309]: Should `formContext` be added to the propTypes?

File: src/CollapsibleField.js (hunk 1 above)
Comment (yusuff-rxnt) [reply src/CollapsibleField.js]: make sense, Updated to send only Table data

**Comment by sneu012** [comment]
Please update the package.json too.

**Comment by sneu012** [comment]
Please update the package.json file

**Review by sneu012 — APPROVED** [review]

**Pull request merged**